app/views/admin/shared/_nested_sidebar.html.erb

```diff
@@ -1,34 +1,44 @@
 <% menu_items.each do |group_name, group| %>
   <li>
     <% if group[:children].length == 1 %>
-      <a class="crayons-link crayons-link--block <%= "crayons-link--current" if deduced_controller(request) == group[:children][0][:controller] %>"
-         href="/admin/<%= group[:children][0][:controller] %>"
-         aria-page="<%= "page" if deduced_controller(request) == group[:children][0][:controller] %>"
-      >
-        <%= inline_svg_tag("#{group[:svg]}", aria: true, class: "dropdown-icon crayons-icon") %>
-        <%= display_name(group_name) %>
-      </a>
+      <% if group[:children][0][:visible] %>
+        <a class="crayons-link crayons-link--block <%= "crayons-link--current" if deduced_controller(request) == group[:children][0][:controller] %>"
+           href="/admin/<%= group[:children][0][:controller] %>"
+           aria-page="<%= "page" if deduced_controller(request) == group[:children][0][:controller] %>"
```

Comment:
I know this PR is already merged, but in a future one, can we change these to `aria-current="page"` - this is the attribute needed to indicate which is the current page to screen reader users (and we shouldn't add the attribute at all if its not the current page) 🙂

Reply:
Oops - I thought I changed that in the previous PR that we worked through. I think it may have gotten reverted with a merge. I'll add this to a subsequent PR